includes/admin/class-wcmp-export.php

```diff
@@ -1441,8 +1441,13 @@ public function addMultiCollo(WC_Order $order, MyParcelCollection $collection, A
         $deliveryOptions     = WCMYPA_Admin::getDeliveryOptionsFromOrder($order);
         $packageType         = $this->getPackageTypeFromOrder($order, $deliveryOptions);
         $isPackage           = AbstractConsignment::PACKAGE_TYPE_PACKAGE_NAME === $packageType;
-        $isMultiColloCountry = in_array($order->get_shipping_country(),
-            [self::COUNTRY_CODE_NL, self::COUNTRY_CODE_BE]
+        $isMultiColloCountry = in_array($order->get_shipping_country(), [self::COUNTRY_CODE_NL, self::COUNTRY_CODE_BE]);
+
+        // Divide the consignment weight by the amount of parcels.
+        $consignment->setPhysicalProperties(
+            [
+                'weight' => (int) ceil($consignment->getTotalWeight() / $colloAmount),
```

Comment:
Volgens mij zorgt ceil er voor dat het al een int gaat worden waardoor je (int) kan weghalen

Reply:
Klinkt logisch, maar `ceil` returnt `float|false` om een of andere reden. Omhoog afronden voelt beter aan voor mij, misschien een keer ietsjes meer betalen en dan geld terug krijgen is beter dan andersom toch? Volgens mij ronden we gewichten vaker naar boven af